src/api/oauth.ts

```diff
@@ -52,15 +52,15 @@ export class OAuth {
    * @returns The URL to authorize a user
    */
   static authorizeUrl(
-    { response_type = "code", redirect_uri, client_id, state, scope }: IAuthorizeUrlParams,
+    { response_type = "code", redirect_uri, client_id, state, scopes }: IAuthorizeUrlParams,
```

Comment:
One breaking change to callout here @dpim is the removal of the `scope` param in favor of supporting a `scopes` param. With `scopes` users can pass an array of the supported scopes into the SDK.

Reply:
I'm not sure if I fully follow - the authorization URI looks for the `scope` query parameter for the list of space delimited scopes (which is standardized as part of [the OAuth spec](https://datatracker.ietf.org/doc/html/rfc6749#section-3.3). 

I would be supportive of offering an optional helper function to pass an array of scope strings that resolves to `?scope=scope1 scope2 scope3`, if that's what you're proposing